src/operator/controllers/intents_reconcilers/port_egress_network_policy/port_egress_network_policy.go

```diff
@@ -51,38 +53,36 @@ func NewPortEgressNetworkPolicyReconciler(
 	}
 }
 
-func (r *PortEgressNetworkPolicyReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-	intents := &otterizev1alpha3.ClientIntents{}
-	err := r.Get(ctx, req.NamespacedName, intents)
-	if k8serrors.IsNotFound(err) {
-		return ctrl.Result{}, nil
+func (r *PortEgressNetworkPolicyReconciler) ReconcileEffectivePolicies(ctx context.Context, eps []effectivepolicy.ServiceEffectivePolicy) (int, error) {
+	currentPolicies := goset.NewSet[types.NamespacedName]()
```

Comment:
Is this only used as a counter?

Reply:
Because this is a set, if you accidentally add the same policy twice, then it will only appear once. But if you did add it twice due to a bug, I think it would cause the policy to have the wrong data, and the set would make it harder to figure out that this is the case (that it was handled twice).

Reply:
Consider using a slice instead